src/makeRobot.js

```diff
@@ -38,7 +38,54 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return 'name: ' + this.name + ', chip version: ' + this.version
+      + ', wheels: ' + this.wheels;
+    },
+    goForward(y = 1) {
+      if (y > 0) {
+        this.coords.y += y;
+      }
+
+      return this;
+    },
+    goBack(y = 1) {
+      if (y > 0) {
+        this.coords.y -= y;
+      }
+
+      return this;
+    },
+    goLeft(x = 1) {
+      if (x > 0) {
+        this.coords.x -= x;
+      }
+
+      return this;
+    },
+    goRight(x = 1) {
+      if (x > 0) {
+        this.coords.x += x;
+      }
+
+      return this;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+  };
 }
```

Comment:
I would also suggest adding empty lines betwwen functions to improve readability
```suggestion
    get info() {
      return 'name: ' + this.name + ', chip version: ' + this.version
      + ', wheels: ' + this.wheels;
    },
    
    goForward(y = 1) {
      if (y > 0) {
        this.coords.y += y;
      }

      return this;
    },
    
    goBack(y = 1) {
      if (y > 0) {
        this.coords.y -= y;
      }

      return this;
    },
    
    goLeft(x = 1) {
      if (x > 0) {
        this.coords.x -= x;
      }

      return this;
    },
    
    goRight(x = 1) {
      if (x > 0) {
        this.coords.x += x;
      }

      return this;
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    evacuate() {
      this.coords.x = 1400;
      this.coords.y = 500;
    },
  };
}
```